t/plugin/clickhouse-logger.t

```diff
@@ -0,0 +1,230 @@
+                                                 })
+            if not ok then
+                ngx.say(err)
+            else
+                ngx.say("passed")
+            end
+        }
+    }
+--- response_body
+property "endpoint_addr" is required
+
+
+
+=== TEST 4: add plugin on routes
+--- yaml_config
+plugins:
+  - clickhouse-logger
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                        "plugins": {
+                            "clickhouse-logger": {
+                                "user": "default",
+                                "password": "a",
+                                "database": "default",
+                                "logtable": "t",
+                                "endpoint_addr": "http://127.0.0.1:8123"
+                            }
+                        },
+                        "upstream": {
+                            "nodes": {
+                                "127.0.0.1:1982": 1
+                            },
+                            "type": "roundrobin"
+                        },
+                        "uri": "/opentracing"
+                }]],
+                [[{
+                    "action":"set",
+                    "node":{
+                        "value":{
+                            "uri":"/opentracing",
+                            "upstream":{
+                                "scheme":"http",
+                                "nodes":{
+                                    "127.0.0.1:1982":1
+                                }
+                            },
+                            "plugins":{
+                                "clickhouse-logger":{
+                                    "batch_max_size":1000,
+                                    "max_retry_count":0,
+                                    "retry_delay":1,
+                                    "ssl_verify":true,
+                                    "endpoint_addr":"http://127.0.0.1:8123",
+                                    "password":"a",
+                                    "buffer_duration":60,
+                                    "timeout":3,
+                                    "user":"default",
+                                    "name":"clickhouse-logger",
+                                    "database":"default",
+                                    "logtable":"t",
+                                    "inactive_timeout":5
+                                }
+                            },
+                            "id":"1"
+                        },
+                        "key":"/apisix/routes/1"
+                    }
+                }]]
+                )
+
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.say(body)
+        }
+    }
+--- request
```

Comment:
We can remove those sections as it is set globally in:
```
if ((!defined $block->error_log) && (!defined $block->no_error_log)) {
        $block->set_value("no_error_log", "[error]");
    }

    if (!defined $block->request) {
        $block->set_value("request", "GET /t");
    }
```

Reply:
Already add on Line 27